### PR TITLE
Fix remaining bugs

### PR DESCRIPTION
**Pull request by wllmwu**
### Administrative Info
Monday Board ID: 853606148
Make sure your branch name conforms to: `<feature/staging/hotfix/...>/[username]/[Monday Item ID]-[3-4 word description separated by dashes]`. Otherwise, please rename your branch and create a new PR.

### Changes
What changes did you make?
- Fade out unanswered badges
- Show badge earned date in detail page
- Remove hardcoded responses
- Show existing values in onboarding dropdowns
- Fix onboarding submission and transitions between app states
- Fix styling so text in settings shows up on Android

### Testing
How did you confirm your changes worked? 
- Tried them a bunch of times

### Confirmation of Change 
Upload a screenshot, if possible. Otherwise, please provide instructions on how to see the change.
- Try it yourself

### CODE REVIEW

**Review by sharmaadvay0 — CHANGES_REQUESTED** [review]
@wllmwu, great work on resolving many of these bugs.

A couple of things that are left:

- The delete and share buttons are still there on the badge page
- The onboarding flow after signing up still throws an error

**Comment by wllmwu** [comment]
> * The delete and share buttons are still there on the badge page
> * The onboarding flow after signing up still throws an error

I will address the first point. Badges are still broken for me so I cannot see any of those pages. (I'm on the `devel` environment.)

I fixed the second point and cannot reproduce it.

**Comment by sharmaadvay0** [comment]
> I fixed the second point and cannot reproduce it.

@wllmwu Maybe try switching to the refactor environment? I'm not sure what else would specifically cause this issue only for me...

**Comment by wllmwu** [comment]
> Maybe try switching to the refactor environment? I'm not sure what else would specifically cause this issue only for me...

@sharmaadvay0 Does it work for you on the devel environment? I was under the impression that we had everything we needed on devel and we no longer need refactor.

**Review by sharmaadvay0 — APPROVED** [review]

File: amplify/hooks/README.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Command Hooks
```
Comment (lynguyxn) [inline amplify/hooks/README.md:1]: should we leave this `amplify/hooks` folder in the repo? idk why it was created

**Review by lynguyxn — APPROVED** [review]
tested the application flows and everything looks great :) just had a minor comment

**Pull request merged**